apps/meteor/lib/getMessageUrlRegex.ts

```diff
@@ -1,2 +1,2 @@
 export const getMessageUrlRegex = (): RegExp =>
-	/([A-Za-z]{3,9}):\/\/([-;:&=\+\$,\w]+@{1})?([-A-Za-z0-9\.]+)+:?(\d+)?((\/[-\+=!:~%\/\.@\,\w]*)?\??([-\+=&!:;%@\/\.\,\w]+)?(?:#([^\s\)]+))?)?/g;
+/(?:\(|'|^)(\w+:\/\/[-A-Za-z0-9\.]+(:\d+)?(\/[-\+=&?:%#\.\/\w]*)?)(?:\)|'|$)|(?:\(|'|^)(https?:\/\/[-A-Za-z0-9\.]+(:\d+)?(\/[-\+=&?:%#\.\/\w]*)?)(?:\)|'|$)/g;
```

Comment:
This regex blocks preview for links with spaces in it's boundaries...

![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/981b7659-b4e1-40ba-9481-b30f3d3ac561)